scan/pki.go

```diff
@@ -94,6 +95,7 @@ func chainValidation(host string) (grade Grade, output Output, err error) {
 		return
 	}
 
+	// This is probably a redundant check
```

Comment:
I didn't dive deep into all the checks [x509#Certificate.VerifyHostname](https://golang.org/pkg/crypto/x509/#Certificate.VerifyHostname) is doing, but I _think_ `cfssl/revoke#VerifyCertificate` is covers a superset of them (obviously adding CRL and OCSP revocation checks).

I think it's probably best to just remove this "quick-fail" method, as it will give less descriptive errors.